pom.xml

```diff
@@ -43,16 +43,72 @@
         <sonar.organization>nashtech-garage</sonar.organization>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
         <sonar.projectKey>nashtech-garage_yas-yas-parent</sonar.projectKey>
-
         <springdoc-openapi-starter-webmvc-ui.version>2.0.2</springdoc-openapi-starter-webmvc-ui.version>
         <org.mapstruct.version>1.4.1.Final</org.mapstruct.version>
+        <org.lombok.version>1.18.30</org.lombok.version>
+        <lombok-mapstruct-binding.version>0.2.0</lombok-mapstruct-binding.version>
+        <logstash-logback-encoder.version>7.4</logstash-logback-encoder.version>
+        <opentelemetry-logback.vesion>1.30.0-alpha</opentelemetry-logback.vesion>
         <spring-cloud.version>2023.0.0</spring-cloud.version>
+        <springdoc-openapi-starter-webmvc-ui.version>2.0.2</springdoc-openapi-starter-webmvc-ui.version>
+        <keycloak-spring-security-adapter.version>21.0.2</keycloak-spring-security-adapter.version>
+        <org.keycloak.version>16.1.0</org.keycloak.version>
+        <commons-validator.version>1.7</commons-validator.version>
+        <org.elasticsearch.version>8.6.2</org.elasticsearch.version>
+        <paypal-checkout-sdk.version>2.0.0</paypal-checkout-sdk.version>
+    </properties>
 
-        <eventuate-platform.version>2023.0.RELEASE</eventuate-platform.version>
+    <dependencies>
```

Comment:
https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&pullRequest=774&id=nashtech-garage_yas-yas-parent&open=AZChSzqn2V8KC0Q2qkrJ

dependencyManagement tag should be before dependencies tag